schema/me/conversation/index.js

```diff
@@ -214,6 +214,33 @@ export const ConversationFields = {
     },
   },
 
+  // If the user is a recipient of the last message, return the relevant delivery id.
+  // This can be used to mark the message as read, or log other events.
+  last_message_delivery_id: {
```

Comment:
We need to grab this in the `Conversation` container fragment. Alternatively, we can bake this in to the mutation code (below).

Essentially, we need to know which 'delivery' to record the 'open' event for. Thus, at the `Conversation` level, we need to make available that relevant id. This is analogous to what we do further up in this file for `is_last_message_to_user`, `last_message_open`, etc. Indeed, they can probably share some code/get refactored. Probably not in this PR tho!

Reply:
Ok, that makes sense. Yes, we should refactor the code for those fields as it's very similar in a future PR.